src/rand.rs

```diff
@@ -81,10 +81,10 @@ fn rand_alphanumerics_impl(rand_fn: &dyn Fn() -> u32, len: usize) -> OsString {
 
 fn rand_bytes_impl(rand_fn: &dyn Fn() -> u32, len: usize) -> Vec<u8> {
     let mut buf: Vec<Vec<u8>> = Vec::new();
-    let mut num = len;
+    let mut num = if len % 4 == 0 { len / 4 } else { len / 4 + 1 };
```

Comment:
How about 
```
let mut num. = (len + 3) >> 2;
```

Reply:
I find the if to be a bit easier to read 😬